README.md

```diff
@@ -303,13 +311,36 @@ when exploring classes and methods.
 
 Autocompletion currently cannot be disabled.
 
+![suggestions](https://user-images.githubusercontent.com/79095619/213680153-38f5df12-8304-480b-aae2-37e6df1433f2.gif)
+
+
+## Modification of the selected lines
+
+Sometimes it is useful to perform some operation on the selected lines.
+The CodeEditor exposes an API for that purpose through the `modifySelectedLines` method inside `CodeController`
+
+```dart
+controller.modifySelectedLines((line) {
+  return line.toUpperCase(); // performs modification on each line
+});
+```
+
+On the basis of the method, basic shortcuts are added to the `CodeEditor`:
+- Indent (Tab)
+- Outdent (Shift-Tab)
+<img src="https://user-images.githubusercontent.com/79095619/213770852-9c0855cb-b8fc-4f40-b33a-380f77c90a0f.gif" width="50%">
+
+- Comment out (Control-/)
+- Uncomment (Control-/)
+<img src="https://user-images.githubusercontent.com/79095619/213770876-829dda06-87b3-4242-9d0e-d26f05090abf.gif" width="50%">
+
+Note: if there is any readonly line inside selection, the method invokation will have no effect.
 
 ## Migration Guides
 
 - [Migrating from code_text_field to 0.1](https://github.com/akvelon/flutter-code-editor/blob/main/doc/migrating/0.1.md)
 - [Migrating from 0.1 to 0.2](https://github.com/akvelon/flutter-code-editor/blob/main/doc/migrating/0.2.md)
 
-
```

Comment:
We had double blank lines before 2-nd level header. Please keep them if there is no major reason to cut them.